library/modules/bigip_device_auth.py

```diff
@@ -320,11 +342,15 @@ def servers(self):
                         "An 'address' field must be provided when specifying separate fields to the 'servers' parameter."
                     )
                 address = server.get('address')
-                port = server.get('port', 49163)
+                if 'port' in server:
+                    port = server.get('port')
```

Comment:
this is unnecessary just define get() method to return None if there is no key e.g. 

`port = server.get('port', None)`

Reply:
Hmmm sure.